web/spec/supabase.yml

```diff
@@ -1391,6 +1391,7 @@ pages:
             .on('UPDATE', handleRecordUpdated)
             .subscribe()
           ```
+          **Note** ``eq`` is a filter that can return events that match the filter.
```

Comment:
@Sonichigo I'm not sure if this helps to clarify `eq`. May I suggest something that references that it'll be a comparison between the same type between the database change of the column vs. the filter value.

Something similar to what I mentioned here: https://github.com/supabase/supabase/issues/4195#issuecomment-1141292215.

Also, how about adding this under `notes` section similar to https://github.com/supabase/supabase/blob/8b207e52dcf3549fbebf0c4a6f6f584d83cc202a/web/spec/supabase.yml#L1399?

Reply:
That makes sense, I'll update it. Totally missed ``notes: | `` Thanks for reminding me.